### PR TITLE
[WIP] Zero delays

### PR DESCRIPTION
**Pull request by lol768**
Opened here mainly for comments and feedback.

Untested at present.

### CODE REVIEW

File: src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java [hunk 1]
```diff
@@ -264,12 +266,27 @@ private void schedule(boolean force) {
...
+                };
+                if (this.client.getMessageDelay() == 0) {
+                    final Runnable zeroDelayRunnable = new Runnable() {
+                        @Override
+                        public void run() {
+                            pollSendRunnable.run();
+                            ClientConnection.this.zeroDelayFuture = ClientConnection.this.channel.eventLoop().submit(this);
+                        }
+                    };
+                    zeroDelayFuture = this.channel.eventLoop().submit(zeroDelayRunnable);
```
Comment (kashike) [inline src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java]: `this.`

Comment (lol768) [inline src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java]: Minor issues like this don't matter if there are fundamental problems with the approach

Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/implementation/NettyManager.java]: rewrite ALL the things!

**Comment by mbax** [comment]
https://github.com/KittehOrg/KittehIRCClientLib/commit/4278fc9d98d4cf1a93f32a105ee841038abc4610

Thanks!

**Comment by lol768** [comment]
Brill, glad to see that code in place

**Pull request closed without merging**